index.php

```diff
@@ -3,14 +3,17 @@
 get_sidebar();
 if ( have_posts() ) : 
 ?>
-
-  <ul>
-    <?php while ( have_posts() ) : the_post(); ?>
-	  <li>
-	    <?php the_content(); ?>
-	  </li>
-    <?php endwhile; ?>
-  </ul>
+  <div class='post-section'>
+	  <ul>
+	      <div class='single-post'>
+		    <?php while ( have_posts() ) : the_post(); ?>
+			  <li>
+			    <?php the_content(); ?>
+			  </li>
+		    <?php endwhile; ?>
+		  </div>  
+	  </ul>
```

Comment:
You dont want to wrap a div around an li. 
It should **always** be `ul > li`.
You are welcome to do `ul > li > div` but never `ul > div > li`.
In this case even `<li class='single-post'>` would be best